conf/messages

```diff
@@ -12,13 +12,15 @@ edit-set.record.edit.former-reference=Former reference - Department (optional)
 edit-set.record.edit.heading=TNA reference: {0}
 edit-set.record.edit.id=PAC-ID: {0}
 edit-set.record.edit.legend=Intellectual properties
+edit-set.record.edit.legal-status=Legal Status
 edit-set.record.edit.scope-and-content=Scope and content
 edit-set.record.edit.start-date=Start date
 edit-set.record.edit.title=Edit record
 edit-set.record.error.end-date-before-start-date=End date cannot precede start date
 edit-set.record.error.end-date=End date is not a valid date
 edit-set.record.error.former-reference-department=Former reference - Department too long, maximum length 255 characters
 edit-set.record.error.scope-and-content=Scope and content too long, maximum length 8000 characters
+edit-set.record.error.legal-status=Select a valid legal status
```

Comment:
Could you keep this list sorted?

Reply:
I don't follow this, do you mean group alpahbetically for errors?

Reply:
The list is now sorted alphabetically which wasn't the case before.